includes/admin/admin-pages.php

```diff
@@ -102,21 +102,16 @@ function give_add_options_links() {
  *
  * @since 2.5.0
  *
- * @global $give_add_ons_page
- *
  * @return void
  */
-function give_add_add_ons_option_link(){
-	global $give_add_ons_page;
+function give_add_add_ons_option_link() {
+	global $submenu;
 
-	//Add-ons
-	$give_add_ons_page = add_submenu_page(
-		'edit.php?post_type=give_forms',
-		esc_html__( 'GiveWP Add-ons', 'give' ),
+	// Add-ons
+	$submenu['edit.php?post_type=give_forms'][] = array(
 		esc_html__( 'Add-ons', 'give' ),
 		'install_plugins',
-		'give-addons',
-		'give_add_ons_page'
+		esc_url( 'https://givewp.com/addons/' ),
```

Comment:
Add a filter here so folks like Astra can easily modify to add an affiliate link.

Reply:
Name of the filter: `give_addon_menu_item_url`.